helm/cas-provision/templates/deployer/deployerRoleBinding.yaml

```
@@ -0,0 +1,15 @@
+kind: RoleBinding
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: {{ .Release.Name }}-shipit-rolebinding
+  labels:
+    release: {{ .Release.Name }}
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: {{ .Release.Namespace }}-deployer
+subjects:
+  - kind: ServiceAccount
+    name: {{ .Release.Name }}-{{.Release.Namespace}}-shipit
+    namespace: {{ .Release.Namespace }}
```

The shipit service account will only be created in the shipit namespace. I'd go with our `-tools` namespace for now, and have a dev/test/prod namespace set for shipit later

```suggestion
    namespace: {{ .Values.deployerNamespace }}
```